packages/compartment-mapper/test/scaffold.js

```diff
@@ -287,6 +289,53 @@ export function scaffold(
     },
   );
 
+  wrap(test.only, 'Bundle')(
+    `${name} / makeArchive / makeSecureBundleFromArchive`,
+    async (t, Compartment) => {
+      t.plan(fixtureAssertionCount);
+      await setup();
+
+      const archive = await makeArchive(readPowers, fixture, {
+        modules,
+        dev: true,
+        policy,
+        tags,
+        searchSuffixes,
+        commonDependencies,
+      });
+      const bundle = await makeSecureBundleFromArchive(
+        {
+          read: async _path => {
+            return archive;
+          },
+        },
+        'fakeLocation',
+        {
+          modules: Object.fromEntries(
```

Comment:
what does specifying the modules here do

Reply:
nothing yet. It doesn't work, so I hacked them away in the fixture. I have a feeling I can fix it